cdviz-collector/src/sources/http.rs

```diff
@@ -87,13 +85,21 @@ async fn health() -> impl IntoResponse {
 //TODO try [deser](https://crates.io/crates/deserr) to return good error
 //TODO use cloudevents
 #[tracing::instrument(skip(app_state, payload))]
-async fn cdevents_collect(
+async fn events_collector(
+    Extension(is_event): Extension<EventType>,
     State(app_state): State<AppState>,
```

Comment:
You declared the Extension as part of AppState, maybe something wrong in the approach.